### PR TITLE
Add a simple `npm test` end-to-end scenario

### PR DESCRIPTION
**Pull request by yvan-sraka**

### CODE REVIEW

**Review by qdrn — APPROVED** [review]

**Comment by qdrn** [comment]
It's failing for Node 12. Do we need to support Node 12 ? 😅

**Comment by yvan-sraka** [comment]
> It's failing for Node 12. Do we need to support Node 12 ? sweat_smile

I would say, no :)

File: package.json [hunk 1]
```diff
@@ -11,5 +11,8 @@
   "dependencies": {
     "yargs": "^17.3.1",
     "mscl-as-transformer": "0.0.1"
+  },
+  "scripts": {
+      "test": "npx massa-sc-create test && cd test && npm run build && npm run clean"
```
Comment (yvan-sraka) [inline package.json:16]: The issue with this test is that it does not test the code pushed into a new PR ...

Comment (qdrn) [reply package.json:16]: It's not perfect but it will catch some things. Do you have something better ?

**Review by qdrn — APPROVED** [review]

**Pull request merged**